docs/cli.md

```diff
@@ -30,50 +30,75 @@ $ sematic run examples/mnist/pytorch
 Since Sematic makes sure not to include unnecessary heavy dependencies, you may
 be prompted to install those needed by the example you want to run.
 
-
 ### User settings
 
-Certain Sematic functionalities need certain settings variables to be set. For
-example, `SnowflakeTable` needs your Snowflake credentials to be able to query
-your data for you.
+Certain Sematic functionalities need certain settings variables to be set.
+These settings control which backend Server will be used when you issue
+commands, and other user-specific configurations. For example, `SnowflakeTable`
+needs your Snowflake credentials to be able to query your data for you.
 
 You can set user settings as follows:
 
 ```shell
-$ sematic settings set VAR VALUE
+$ sematic user-settings set VAR VALUE
 ```
 
-For example
+For example:
 
 ```shell
-$ sematic settings set SNOWFLAKE_USER "foobar"
+$ sematic user-settings set SNOWFLAKE_USER "foobar"
 ```
 
-Then you can check your stored settings with
+Then you can check your stored settings with:
 
 ```shell
-$ sematic settings show
-Active settings:
+$ sematic user-settings show
+Active user-settings:
 
 SNOWFLAKE_USER: foobar
 ```
 
-These settings are simply store in the `/.sematic/settings.yaml` file on
+These settings are simply stored in the `~/.sematic/settings.yaml` file on
 your machine.
 
 You can always override them at runtime with an environment variable:
 
 ```shell
-$ SNOWFLAKE_USER="notfoobar" sematic settings show
-Active settings:
+$ SNOWFLAKE_USER="notfoobar" sematic user-settings show
+Active user settings:
 
 SNOWFLAKE_USER: notfoobar
 ```
 
 If you'd like, you can even change which directory Sematic uses to hold its
 settings by setting the environment variable `SEMATIC_CONFIG_DIR`. This can
-be either a relative path (it will be treated as relative to your home directory)
-or an absolute path.
+be either a relative path (it will be treated as relative to your home
+directory) or an absolute path.
+
+### Server settings
+
+The same way that certain settings need to be kept on the user side, certain
+ones need to configure the backend Server.
+
+These can be accessed and modified just like the user settings, but by using
+the `sematic user-settings` command instead:
+
+```shell
+$ sematic server-settings show
+Active server settings:
+
+AWS_S3_BUCKET: sematic-dev
+KUBERNETES_NAMESPACE: default
+SEMATIC_AUTHORIZED_EMAIL_DOMAIN: sematic.ai
+SEMATIC_AUTHENTICATE: 'true'
+GOOGLE_OAUTH_CLIENT_ID: XXX
+GRAFANA_PANEL_URL: XXX
+
+```
```

Comment:
I actually don't think it's necessary to have the CLI support setting server settings. The server settings will only be used where the server is deployed, where it is acceptable to simply edit the yaml file.
Granted, the server can also run locally (`$ sematic start`) but maybe we can expose only user settings through the CLI and let people edit the server yaml by hand? Then we could keep the `$ sematic settings set` syntax.

Curious to hear what @augray thinks.

Reply:
I like having the ability to at least *view* the server settings from the CLI; it will give a way sanity check that the server settings are getting picked up as expected. That said, user settings will be the thing you're using the CLI for 99% of the time. So how about this:
```
sematic settings show  # shows user settings
sematic settings show  --server # shows server settings
```

Reply:
Personally it's less about the code changes being heavy, and more about the ergonomics being more heavy. Commands that are run more often should be shorter. If 99% of the time people want to see user settings, we should prefer `sematic settings show` for that rather than `sematic user-settings show`. `--server` is just a suggestion for this. You could also do this:
```
sematic settings show  # shows user settings
sematic server-settings show  # shows server settings
```

Reply:
Agree with @augray here regarding naming.

My original point is that it is unlikely that users will have access to the CLI where the server is deployed.

Take our example:
- on the web box, we deploy the docker image. The CLI is not available outside the docker image, since the pip wheel is not installed, and we (usually) do not go into the docker container. We parametrize the server with env vars passed to the container.
- in k8s, we use a helm chart with env vars

So where would we use the CLI to check server settings?

Reply:
The server settings CLI will be used by people running the Sematic server locally, like members of out open source community, or developers.

I renamed the cli commands to `sematic settings` and `sematic server-settings` respectively, and the files to `~/.sematic/settings.yaml` and `~/.sematic/server.yaml`, respectively, as requested in these comments.